clang-tools-extra/docs/clang-tidy/checks/readability/use-builtin-literals.rst

```diff
@@ -0,0 +1,36 @@
+.. title:: clang-tidy - readability-use-builtin-literals
+
+readability-use-builtin-literals
+================================
+
+Finds literals explicitly casted to a type that could be expressed using builtin prefixes or suffixes.
+
+.. code-block: c++
+
+    (char16_t)U'a'; // -> u'a'
```

Comment:
Shouldn't this be `(char16_t)'a'`?

Reply:
Originally I put it there to illustrate that superfluous sequences will be stripped before the correct one is added. I swapped it with the second one to make it less jarring.